cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java

```diff
@@ -80,42 +81,61 @@ public Integer call() throws Exception {
         @Option(names = { "-f", "--force" }, required = false, description = "Force deletion without confirmation.")
         Boolean force;
 
+        private Map<String, ServiceBean> readServicesFromYamlFile() throws IOException {
+            SpecBean specBean = YamlMapper.loadBean(yamlCommandOptions.getYamlFilePath(), SpecBean.class);
```

Comment:
So far, good, but I would lower the method further for documentation purposes.

Should be around line 128, above the private method:
doRemoveServiceInstance().

Then you have the:

Options

Overrides

private methods